LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 IPFS
```

Comment:
Fair. It's actually attributed in a lot of these. Let's discuss in https://github.com/ipfs/community/issues/139.